libs/data-mapper/src/lib/core/state/DataMapSlice.ts

```diff
@@ -726,32 +736,18 @@ export const deleteNodeWithKey = (curDataMapState: DataMapState, reactFlowKey: s
 
     curDataMapState.curDataMapOperation.selectedItemKey = undefined;
     doDataMapOperation(curDataMapState, { ...curDataMapState.curDataMapOperation, currentFunctionNodes: newFunctionsState });
+
     curDataMapState.notificationData = {
       type: NotificationTypes.FunctionNodeDeleted,
       autoHideDurationMs: deletedNotificationAutoHideDuration,
     };
+
     return;
   }
 
   // Item to be deleted is a connection
   const connections = { ...curDataMapState.curDataMapOperation.dataMapConnections };
 
-  /*
-    const sourceId = getSourceIdFromReactFlowConnectionId(reactFlowKey);
-    const sourceSchema = curDataMapState.curDataMapOperation.flattenedSourceSchema;
-    const canDelete = canDeleteConnection(sourceId, sourceSchema);
```

Comment:
I can't recall if we use canDeleteConnection() anywhere else - if not, might be good to go ahead and delete that too (and its associated test, again IIRC)

Reply:
Can do